src/components/Person/Person.jsx

```diff
@@ -1 +1,19 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+    {person.age !== undefined && (
+      <>
+        <p className="Person__age">{`I am ${person.age}`}</p>
+      </>
+    )}
+    {person.isMarried === false ? (
+      <p className="Person__partner">I am not married</p>
+    ) : (
+      (person.sex === 'm' && (
+        <p className="Person__partner">{`${person.partnerName} is my wife`}</p>
+      )) || (
+        <p className="Person__partner">{`${person.partnerName} is my husband`}</p>
```

Comment:
Let's simplifi it a bit and create a variable for person partner :
`const partner = person.sex === 'm' ? 'wife' : 'husband'`

And you can also have just one <p> tag and change only it's content